### PR TITLE
Fix border/padding while loading jt edit form

### PR DESCRIPTION
**Pull request by keithjgrant**
Addresses #4614

### CODE REVIEW

File: awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import {
   Form,
   FormGroup,
   Card,
+  CardBody,
```
Comment (marshmalien) [inline awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx]: We can remove Card and CardBody imports now that we're not using them.

**Review by marshmalien — CHANGES_REQUESTED** [review]

File: awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx (hunk 1 above)
Comment (keithjgrant) [reply awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx]: 👍🏻

**Review by marshmalien — APPROVED** [review]
👍

**Comment by keithjgrant** [comment]
recheck

**Comment by jakemcdermott** [comment]
recheck

**Comment by jakemcdermott** [comment]
recheck

**Comment by jakemcdermott** [comment]
recheck

**Pull request merged**